cloud/ec2_util.go

```diff
@@ -408,6 +409,18 @@ func makeBlockDeviceMappings(mounts []MountPoint) ([]*ec2aws.BlockDeviceMapping,
 			if mount.Iops != 0 {
 				m.Ebs.Iops = aws.Int64(mount.Iops)
 			}
+
+			if mount.Throughput != 0 {
+				//aws only allows values between 125 and 1000
+				if mount.Throughput > 1000 || mount.Throughput < 125 {
```

Comment:
nit: 1000 (no ,)

Reply:
I don't think there is a (,), you may have seen an older version of my changes maybe?

Reply:
sorry this is on the wrong line; I meant for line 421

Reply:
done